Config/armbsp.cfg

```diff
@@ -0,0 +1,33 @@
+[gdbServer]
+address = %(ARMBSP_RASPI_ADDRESS)s
```

Comment:
Why ARMBSP?

Reply:
Oh... I see. This is the second config.

Reply:
I was using this configuration to connect to armbsp setup when taste-setup was not available.